### PR TITLE
lib: bsdlib: Updated bsd_init() to use new bsdlib version

### PR DESCRIPTION
**Pull request by tommi-m**
Added trace and memory parameters.

Signed-off-by: Tommi Mammela <tommi.mammela@nordicsemi.no>

### CODE REVIEW

File: lib/bsdlib/bsdlib.c [hunk 1]
```diff
@@ -28,8 +28,14 @@ static int _bsdlib_init(struct device *unused)
 	IRQ_DIRECT_CONNECT(BSD_NETWORK_IRQ, BSD_NETWORK_IRQ_PRIORITY,
 			   ipc_proxy_irq_handler, 0);
 
-	init_ret = bsd_init();
+	const bsd_init_params_t init_params =
+	{
+		.trace_on = true,                   //TODO: get the right definition from Kconfig
+		.bsd_memory_address = 0x20010000,   //TODO: get the right definition from partition manager
+		.bsd_memory_size = 0xc020           //TODO: get the right definition from partition manager
```
Comment (tejlmand) [inline lib/bsdlib/bsdlib.c:35]: I assume you will update before we merge this ?

Comment (tommi-m) [reply lib/bsdlib/bsdlib.c]: It depends when partition manager update is ready. Before that we need to pass these default values to bsd_init().

Comment (tejlmand) [reply lib/bsdlib/bsdlib.c]: got it. thanks for the info.

**Comment by tejlmand** [comment]
depends on: https://github.com/NordicPlayground/nrfxlib/pull/165

**Comment by rlubos** [comment]
Please update the SHA: 3eed8cbd8f6b0fa38aaa6b7d5d0e33bd7408f0d1

**Pull request merged**